src/components/NavBar.tsx

```diff
@@ -71,19 +71,16 @@ export default function NavBar() {
                           </Link>
                         </NavigationMenuLink>
                       </li>
-                      <Link href="/store">
+                      <Link href="/store/wraps">
                         <ListItem title="Shop">
                           View the store.
                         </ListItem>
                       </Link>
                       <Link href="/">
```

Comment:
This isn't used, this will still appear in the navbar. You only deleted the ListItem but not the link itself.